test-project/src/test/java/com/redhat/devtools/intellij/commonUiTestLibrary/fixturesTest/dialogs/navigation/ProjectStructureAndTipDialogs.java

```diff
@@ -8,68 +9,74 @@
  * Contributors:
  * Red Hat, Inc. - initial API and implementation
  ******************************************************************************/
-package com.redhat.devtools.intellij.commonUiTestLibrary.fixturesTest.dialogs.navigation;
-
 import com.intellij.remoterobot.utils.WaitForConditionTimeoutException;
 import com.redhat.devtools.intellij.commonUiTestLibrary.LibraryTestBase;
 import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.mainIdeWindow.MainIdeWindow;
 import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.mainIdeWindow.ideStatusBar.IdeStatusBar;
-import com.redhat.devtools.intellij.commonUiTestLibrary.utils.testExtension.ScreenshotAfterTestFailExtension;
 import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.dialogs.information.TipDialog;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
+import com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.dialogs.information.ProjectStructureDialog;
+import org.junit.jupiter.api.*;
 
 import java.time.Duration;
 
 import static com.redhat.devtools.intellij.commonUiTestLibrary.fixtures.dialogs.information.TipDialog.closeTipDialogIfItAppears;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 /**
- * SearchEverywherePopup and TipDialog test
+ * TipDialog test and ProjectStructure test via SearchEverywherePopup
  *
  * @author zcervink@redhat.com
  */
-@ExtendWith(ScreenshotAfterTestFailExtension.class)
-public class SearchEverywherePopupAndTipDialogTest extends LibraryTestBase {
-    private final String projectName = "tip_dialog_java_project";
+public class ProjectStructureAndTipDialogs extends LibraryTestBase {
+    private static final String projectName = "tip_dialog_java_project";
 
-    @BeforeEach
-    public void prepareProject() {
+    @BeforeAll
+    public static void prepareProject() {
         createNewProject(projectName, "Java");
     }
 
-    @AfterEach
-    public void closeProject() {
-        super.closeProject();
+    @AfterAll
+    public static void closeProject() {
+        LibraryTestBase.closeProject();
+    }
+
```

Comment:
I suggest to remove 1 empty line and keep only 1.

Reply:
done